user/languages/r.md

```diff
@@ -206,6 +206,10 @@ repos:
   on this one. This can be quite expensive, so it's not recommended to leave
   this set to `true`.
 
+* `disable_homebrew`: if `true` this removes the preinstalled homebrew 
+  installation on OS X. Useful to test if the package builds on a vanilla OS X
+  machine, such as the CRAN mac builder.
```

Comment:
It does add value because R package authors specifically want to test if the package will build on the CRAN servers, which are the main distribution network. The CRAN website is not too interesting, most people only know CRAN via R's built-in package manger. But we could add a link.